Include/scripts/groovy/com/kms/katalon/core/reporting/basic/reporting/JsStepModel.java

```diff
@@ -203,8 +201,14 @@ private String getLogFolder(MessageLogRecord messageLog) {
     }
 
     private String encodeFileContent(final File file) throws FileNotFoundException, Exception, IOException {
-        try (InputStream is = new FileInputStream(file)) {
+        InputStream is = null;
```

Comment:
The old code already close the InputStream after the method has done. It's `try-with-resource`, please revert to the old code if we just want to close the stream.

Reply:
Use this way to improve the image number attached on the report